src/util/format.ts

```diff
@@ -215,6 +216,11 @@ export function getTooltipMarker(inOpt: ColorString | GetTooltipMarkerOpt, extra
     const type = opt.type;
     extraCssText = opt.extraCssText;
     const renderMode = opt.renderMode || 'html';
+    let size = opt.size;
```

Comment:
Where does `size` pass into function, Did you forget to commit `tooltipMarkup.ts`?

Reply:
i will add this as a parameter.